package/src/components/MessageInput/components/NativeAttachmentPicker.tsx

```diff
@@ -103,7 +118,19 @@ export const NativeAttachmentPicker = ({
     }).start(onRequestedClose);
   };
 
-  const buttons = [];
+  // do not allow poll creation in threads
+  const buttons =
+    threadList && hasCreatePoll && ownCapabilities.sendPoll
```

Comment:
I thought about that too, but that's not 100% correct - as what happens if `hasCreatePoll` is specifically passed as `true` and yet the `sendPoll` capability is `false` ? It will be overridden and the capability will not be respected.

Of course, they can figure this out from the dashboard too but it would be very confusing as well as a source of bugs too.

Reply:
Well I think we ignore this case. Anyone would be anyways end up getting no polls if they do that. Dashboard config in anyways has to be given priority over local config

Reply:
Well not quite, we can't - let's say that for example someone wants the capabilities there but maybe disable poll creation for 30 minutes or so, if this relies entirely on the capability they wouldn't be able to do that. Besides, these should be kept separate in my opinion since they mean 2 different things (one is the capability to send a poll, the other is whether we want someone to send a poll)